.github/workflows/dependabot-ci.yml

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: dependabot-ci
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+concurrency:
+  group: dependabot-ci-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+jobs:
+  dependabot-check:
+    if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request'}}
+    runs-on: ubuntu-20.04
+    env:
+      CI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      NODE_VERSION: 16.14.2
```

Comment:
So we are using Node v16 in our main build.
https://github.com/apache/ozone/blob/e4ff5fa1b9b6a8335e5e1d7a118c9c3d4b46281d/hadoop-ozone/recon/pom.xml#L106
The reason we are sticking with it is because Node v17 and above breaks Recon. We would need to do `react-scripts eject`, and then setup webpack and babel configs, and move away from react-app-rewired, so that we can better control on the builds and figure out the error being thrown.
Node V17 and above has an error with internal webpack dependency clashing, hence the need for eject.

Reply:
Can you file a Jira for this if we don't have one already? We don't have to handle this now and you can leave it unassigned, but it would be good to have this documented as a task.

Reply:
I created [HDDS-9639](https://issues.apache.org/jira/browse/HDDS-9639) for tracking this task. Assigned it to myself as I am mostly completed with the major migration of React code to functional components. Experimenting with the other upgrades.